internal/wasm/sdk/test/e2e/external_test.go

```diff
@@ -0,0 +1,217 @@
+// Copyright 2020 The OPA Authors.  All rights reserved.
+// Use of this source code is governed by an Apache2
+// license that can be found in the LICENSE file.
+
+package e2e
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/open-policy-agent/opa/ast"
+	"github.com/open-policy-agent/opa/internal/wasm/sdk/opa"
+	"github.com/open-policy-agent/opa/rego"
+	"github.com/open-policy-agent/opa/test/cases"
+	"github.com/open-policy-agent/opa/types"
+	"github.com/open-policy-agent/opa/util"
+)
+
+var caseDir = flag.String("case-dir", "../../../../../test/cases/testdata", "set directory to load test cases from")
+var exceptionsFile = flag.String("exceptions", "./exceptions.yaml", "set file to load a list of test names to exclude")
+
+var exceptions map[string]string
+
+func TestMain(m *testing.M) {
+	exceptions = map[string]string{}
+
+	bs, err := ioutil.ReadFile(*exceptionsFile)
+	if err != nil {
+		fmt.Println("Unable to load exceptions file: " + err.Error())
+		os.Exit(1)
+	}
+	err = util.Unmarshal(bs, &exceptions)
+	if err != nil {
+		fmt.Println("Unable to parse exceptions file: " + err.Error())
+		os.Exit(1)
+	}
+
+	addTestSleepBuiltin()
+
+	os.Exit(m.Run())
+}
+
+func TestWasmE2E(t *testing.T) {
+
+	ctx := context.Background()
+
+	for _, tc := range cases.MustLoad(*caseDir).Sorted().Cases {
+		if tc.Note != "withkeyword/invalidate comprehension cache" {
```

Comment:
I think this ought to be the opposite. I'm also wondering if we should just have an exception for this instead. The latter isn't too important.

Reply:
This check was from a last minute debugging thing and I didn't push the updated changes 😅  updated in latest to not include it.